scripts/bash/getDeployablePackages.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash -e
 
-echo "admin-panel" "lesmis" "psss" "datatrak-web" "tupaia-web" "web-frontend" "central-server" "data-table-server" "datatrak-web-server" "entity-server" "lesmis-server" "meditrak-app-server" "psss-server" "report-server" "tupaia-web-server" "web-config-server" "admin-panel-server" # admin-panel-server last as it depends on report-server
+# meditrak-app-server before central-server so that the meditrak-sync-queue is listening before we run the migrations
```

Comment:
Hmm, I don't love the way this implies servers need to start in a specific order. IMO servers should be smart enough to not have to be run in any specific order. The start order is also not guaranteed in general. 

They probably do start in this order the first time when these are started with `pm2 start x` but not subsequently when the machine starts again and loads the pm2 save file. And in the future if we move to a pure pm2 config file setup/or another similar infrastructure by declaration, then everything will start at the same time.

Reply:
Discussed, will sort out in a separate PR.